src/Commands/WaContent.php

```diff
@@ -58,7 +60,7 @@ public function import($args, $assocArgs)
         $this->disableHooks(); // Disable various hooks and filters during import
         $this->fixNonceErrors();
 
-        $this->repository = new ContentRepository();
+        $this->repository = new ContentRepository($assocArgs['locale'] ?? null);
```

Comment:
It is possible to add defaults to `assocArgs`
https://make.wordpress.org/cli/handbook/commands-cookbook/#annotating-with-phpdoc

Reply:
Thats nice, but I just tested, and you cant default to null 😢